src/cascadia/TerminalApp/CommandPalette.cpp

```diff
@@ -804,8 +804,8 @@ namespace winrt::TerminalApp::implementation
     // - <unused>
     // Return Value:
     // - <none>
-    void CommandPalette::_filterTextChanged(IInspectable const& /*sender*/,
-                                            Windows::UI::Xaml::RoutedEventArgs const& /*args*/)
+    void CommandPalette::_filterTextChanged(const IInspectable& /*sender*/,
```

Comment:
>const IInspectable& [](http://example.com/codeflow?start=44&length=19)

Random place to ask this, but will the `const T&` thing be enforced in the formatter? IIRC the `T const&` style comes from the auto-generated cppwinrt code, so it's entirelly likely that this leaks back into the project in the future.

Reply:
k cool